### PR TITLE
Added India UDISE geography

### PR DESCRIPTION
**Pull request by thejeshgn**
@pradh PR for importing UDISE specific Geographies

---
This change is [Reviewable](https://reviewable.io/reviews/datacommonsorg/data/413)

### CODE REVIEW

**Review by pradh — CHANGES_REQUESTED** [review]
Thanks for the PR.  Main feedback is around the TMCF formats, otherwise LG.

> Reviewed 7 of 22 files at r1.
> *[Reviewable](https://reviewable.io/reviews/datacommonsorg/data/413#-:-MY6ez_jDFlysp4iZTfk:b-t9eah)* status: 7 of 22 files reviewed, 13 unresolved discussions (waiting on @thejeshgn)

___
*[scripts/india_udise/udise_geography/preprocess.py, line 14 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dlleAFzhRhsUs8oI:-MY6dlleAFzhRhsUs8oJ:b-r9njky) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/preprocess.py#L14)):*
> ```
> Classes and methods to import Average wage/salary earnings from Periodic Labour Force Survey (PLFS)
> ```

Change to UDISE?
___
*[scripts/india_udise/udise_geography/preprocess.py, line 83 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dr_25J4bWGL5VjCs:-MY6dr_25J4bWGL5VjCt:bsahbvx) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/preprocess.py#L83)):*
<details><summary><i alt="summary">Quoted 60 lines of code…</i></summary>

> ```
> INDIA_ISO_CODES = {
>     "Andhra Pradesh": "IN-AP",
>     "Arunachal Pradesh": "IN-AR",
>     "Assam": "IN-AS",
>     "Bihar": "IN-BR",
>     "Chattisgarh": "IN-CT",
>     "Chhattisgarh": "IN-CT",
>     "Goa": "IN-GA",
>     "Gujarat": "IN-GJ",
>     "Haryana": "IN-HR",
>     "Himachal Pradesh": "IN-HP",
>     "Jharkhand": "IN-JH",
>     "Jharkhand#": "IN-JH",
>     "Karnataka": "IN-KA",
>     "Kerala": "IN-KL",
>     "Madhya Pradesh": "IN-MP",
>     "Madhya Pradesh#": "IN-MP",
>     "Maharashtra": "IN-MH",
>     "Manipur": "IN-MN",
>     "Meghalaya": "IN-ML",
>     "Mizoram": "IN-MZ",
>     "Nagaland": "IN-NL",
>     "Nagaland#": "IN-NL",
>     "Odisha": "IN-OR",
>     "Punjab": "IN-PB",
>     "Rajasthan": "IN-RJ",
>     "Sikkim": "IN-SK",
>     "Tamil Nadu": "IN-TN",
>     "Tamilnadu": "IN-TN",
>     "Telengana": "IN-TG",
>     "Telangana": "IN-TG",
>     "Tripura": "IN-TR",
>     "Uttarakhand": "IN-UT",
>     "Uttar Pradesh": "IN-UP",
>     "West Bengal": "IN-WB",
>     "Andaman and Nicobar Islands": "IN-AN",
>     "Andaman & Nicobar Islands": "IN-AN",
>     "Andaman & N. Island": "IN-AN",
>     "A & N Islands": "IN-AN",
>     "Chandigarh": "IN-CH",
>     "Dadra and Nagar Haveli": "IN-DN",
>     "Dadra & Nagar Haveli": "IN-DN",
>     "Dadar Nagar Haveli": "IN-DN",
>     "Daman and Diu": "IN-DD",
>     "Daman & Diu": "IN-DD",
>     "Delhi": "IN-DL",
>     "Jammu and Kashmir": "IN-JK",
>     "Jammu & Kashmir": "IN-JK",
>     "Ladakh": "IN-LA",
>     "Ladhak": "IN-LA",
>     "Lakshadweep": "IN-LD",
>     "Lakshwadeep": "IN-LD",
>     "Pondicherry": "IN-PY",
>     "Puducherry": "IN-PY",
>     "Puduchery": "IN-PY",
>     "Dadra and Nagar Haveli and Daman and Diu": "IN-DN_DD",
>     "all India": "IN",
>     "all-India": "IN",
>     "ALL INDIA": "IN",
> }
> ```
</details>


Not necessary to address in this PR. Just curious if these maps from the various imports all the same?  If so, it might be worth sharing them from a common library (e.g., in https://github.com/datacommonsorg/data/tree/master/util).
___
*[scripts/india_udise/udise_geography/preprocess.py, line 168 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6eYo94YdSt3RHb_eK:-MY6eYo94YdSt3RHb_eL:b-8vv8pf) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/preprocess.py#L168)):*
> ```Python
>             json_data_file = open(self.states_json_data_file_path, "w")
>             json_data_file.write(json.dumps(response.json()))
>             json_data_file.close()
> ```

else: fail loudly?  (elsewhere too)
___
*[scripts/india_udise/udise_geography/README.md, line 7 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dbxuCoV0UqiA9OCS:-MY6dbxuCoV0UqiA9OCT:b-pclpbg) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/README.md#L7)):*
> ```
> Available for download using API.
> ```

Could we say a bit more about the API being used to fetch this data? Any pointers.
___
*[scripts/india_udise/udise_geography/UDISE_Blocks.tmcf, line 1 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6a_4fDv9kfNMwRN2h:-MY6a_4fDv9kfNMwRN2i:b5oadfa) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Blocks.tmcf#L1)):*
> ```
> Node: dcid:udiseCode/C:UDISE_Blocks->udise_block_code
> ```

Unfortunately, the tmcf format doesn't support this yet.

This might need to be:

```
Node: E:UDISE_Blocks->E1
dcid: C:UDISE_Blocks->udise_block_code
```

where, udise_block_code column has "udiseCode/XYZ".
___
*[scripts/india_udise/udise_geography/UDISE_Blocks.tmcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6awK_7rYSB9Dgygwu:-MY6awK_7rYSB9Dgygwv:bh1suz4) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Blocks.tmcf#L5)):*
> ```
> dcid:udiseCode/C:UDISE_Blocks->udise_district_code
> ```

Here too, this might need to be:

```
containedInPlace: C:UDISE_Blocks->udise_district_code
```

udise_district_code is "dcid:udiseCode/XYZ". Because it is a reference.
___
*[scripts/india_udise/udise_geography/UDISE_Districts.csv, line 2 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6bOht2_A6o1dwi-YR:-MY6bOht2_A6o1dwi-YS:b-usb0w6) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Districts.csv#L2)):*
> ```
> 36,3601
> ```
Per comment above, this might need to change to prefix udiseCode/
___
*[scripts/india_udise/udise_geography/UDISE_Districts.csv, line 3 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6bVLUCNqi_M8cfp9Q:-MY6bVLUCNqi_M8cfp9R:b-5rmssb) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Districts.csv#L3)):*
> ```
> AGAR MALWA
> ```

Could we make this Title Case 
___
*[scripts/india_udise/udise_geography/UDISE_Districts.tmcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cJiu94BB-5XRfHJE:-MY6cJiu94BB-5XRfHJF:bme5zo2) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Districts.tmcf#L5)):*
> ```
> Node: dcid:udiseCode/C:UDISE_Districts->udise_district_code
> ```

Same comment as above.
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 1 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6d81Q6NpjRXi89yZO:-MY6d81Q6NpjRXi89yZP:bpw3lgd) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L1)):*
> ```
> Node: dcid:UDISEDistrict
> ```

This could be added to https://github.com/datacommonsorg/schema/blob/main/places.mcf
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cXul54HniUFPG-u_:-MY6cXul54HniUFPG-ua:b3lmnfg) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L5)):*
> ```
> Education
> ```

nit: Educational ?
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cj2K7z4sM2udm1v1:-MY6cj2K7z4sM2udm1v2:bllfsb4) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L5)):*
> ```
> UDISE
> ```

Unified District Information System for Education (UDISE)
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 11 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cvUC7s6qW3PX_x2J:-MY6cvUC7s6qW3PX_x2K:b-svdtoj) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L11)):*
> ```
> subClassOf: schema:AdministrativeArea
> name: "UDISEBlock"
> description: "Educational block that covers multiple schools."
> ```

"Educational block in India as per the Unified District Information System for Education (UDISE). A UDISEBlock covers multiple schools."

**Review by thejeshgn — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/datacommonsorg/data/413)* status: 3 of 22 files reviewed, 12 unresolved discussions (waiting on @pradh)

___
*[scripts/india_udise/udise_geography/preprocess.py, line 14 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dlleAFzhRhsUs8oI:-MY84zHs2YGo7QnYzrw2:b32w0x) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/preprocess.py#L14)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> Classes and methods to import Average wage/salary earnings from Periodic Labour Force Survey (PLFS)
> ```

Change to UDISE?
</blockquote></details>

OK
___
*[scripts/india_udise/udise_geography/preprocess.py, line 83 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dr_25J4bWGL5VjCs:-MY85P6TDIPX3JCUMybK:b2nha9w) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/preprocess.py#L83)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> INDIA_ISO_CODES = {
>     "Andhra Pradesh": "IN-AP",
>     "Arunachal Pradesh": "IN-AR",
>     "Assam": "IN-AS",
>     "Bihar": "IN-BR",
>     "Chattisgarh": "IN-CT",
>     "Chhattisgarh": "IN-CT",
>     "Goa": "IN-GA",
>     "Gujarat": "IN-GJ",
>     "Haryana": "IN-HR",
>     "Himachal Pradesh": "IN-HP",
>     "Jharkhand": "IN-JH",
>     "Jharkhand#": "IN-JH",
>     "Karnataka": "IN-KA",
>     "Kerala": "IN-KL",
>     "Madhya Pradesh": "IN-MP",
>     "Madhya Pradesh#": "IN-MP",
>     "Maharashtra": "IN-MH",
>     "Manipur": "IN-MN",
>     "Meghalaya": "IN-ML",
>     "Mizoram": "IN-MZ",
>     "Nagaland": "IN-NL",
>     "Nagaland#": "IN-NL",
>     "Odisha": "IN-OR",
>     "Punjab": "IN-PB",
>     "Rajasthan": "IN-RJ",
>     "Sikkim": "IN-SK",
>     "Tamil Nadu": "IN-TN",
>     "Tamilnadu": "IN-TN",
>     "Telengana": "IN-TG",
>     "Telangana": "IN-TG",
>     "Tripura": "IN-TR",
>     "Uttarakhand": "IN-UT",
>     "Uttar Pradesh": "IN-UP",
>     "West Bengal": "IN-WB",
>     "Andaman and Nicobar Islands": "IN-AN",
>     "Andaman & Nicobar Islands": "IN-AN",
>     "Andaman & N. Island": "IN-AN",
>     "A & N Islands": "IN-AN",
>     "Chandigarh": "IN-CH",
>     "Dadra and Nagar Haveli": "IN-DN",
>     "Dadra & Nagar Haveli": "IN-DN",
>     "Dadar Nagar Haveli": "IN-DN",
>     "Daman and Diu": "IN-DD",
>     "Daman & Diu": "IN-DD",
>     "Delhi": "IN-DL",
>     "Jammu and Kashmir": "IN-JK",
>     "Jammu & Kashmir": "IN-JK",
>     "Ladakh": "IN-LA",
>     "Ladhak": "IN-LA",
>     "Lakshadweep": "IN-LD",
>     "Lakshwadeep": "IN-LD",
>     "Pondicherry": "IN-PY",
>     "Puducherry": "IN-PY",
>     "Puduchery": "IN-PY",
>     "Dadra and Nagar Haveli and Daman and Diu": "IN-DN_DD",
>     "all India": "IN",
>     "all-India": "IN",
>     "ALL INDIA": "IN",
> }
> ```

Not necessary to address in this PR. Just curious if these maps from the various imports all the same?  If so, it might be worth sharing them from a common library (e.g., in https://github.com/datacommonsorg/data/tree/master/util).
</blockquote></details>

ACK
___
*[scripts/india_udise/udise_geography/preprocess.py, line 168 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6eYo94YdSt3RHb_eK:-MY85QvM9mzMDVOcyC8q:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/preprocess.py#L168)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

else: fail loudly?  (elsewhere too)
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/README.md, line 7 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dbxuCoV0UqiA9OCS:-MY85U5XCR0C0AMo75RN:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/README.md#L7)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> Available for download using API.
> ```

Could we say a bit more about the API being used to fetch this data? Any pointers.
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Blocks.tmcf, line 1 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6a_4fDv9kfNMwRN2h:-MY85V293GvLkm6JWOBR:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Blocks.tmcf#L1)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> Node: dcid:udiseCode/C:UDISE_Blocks->udise_block_code
> ```

Unfortunately, the tmcf format doesn't support this yet.

This might need to be:

```
Node: E:UDISE_Blocks->E1
dcid: C:UDISE_Blocks->udise_block_code
```

where, udise_block_code column has "udiseCode/XYZ".
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Blocks.tmcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6awK_7rYSB9Dgygwu:-MY85VS61Y0NZgMPyx1l:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Blocks.tmcf#L5)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> dcid:udiseCode/C:UDISE_Blocks->udise_district_code
> ```

Here too, this might need to be:

```
containedInPlace: C:UDISE_Blocks->udise_district_code
```

udise_district_code is "dcid:udiseCode/XYZ". Because it is a reference.
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Districts.csv, line 2 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6bOht2_A6o1dwi-YR:-MY85WMC7Ry2euSJ1D_1:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Districts.csv#L2)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> 36,3601
> ```
Per comment above, this might need to change to prefix udiseCode/
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Districts.csv, line 3 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6bVLUCNqi_M8cfp9Q:-MY85Wv93TRKK5Lz83EJ:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Districts.csv#L3)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> AGAR MALWA
> ```

Could we make this Title Case 
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Districts.tmcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cJiu94BB-5XRfHJE:-MY85YijAtGxfknyvErZ:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Districts.tmcf#L5)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> Node: dcid:udiseCode/C:UDISE_Districts->udise_district_code
> ```

Same comment as above.
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 1 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6d81Q6NpjRXi89yZO:-MY85aAh4OZ5-Ae3ZaCS:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L1)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

This could be added to https://github.com/datacommonsorg/schema/blob/main/places.mcf
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cj2K7z4sM2udm1v1:-MY85_8E9SijOGB_IiLv:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L5)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> UDISE
> ```

Unified District Information System for Education (UDISE)
</blockquote></details>

Done.
___
*[scripts/india_udise/udise_geography/UDISE_Places.mcf, line 11 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6cvUC7s6qW3PX_x2J:-MY85_gk9GKEeW4_GQm8:b-896fix) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Places.mcf#L11)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

"Educational block in India as per the Unified District Information System for Education (UDISE). A UDISEBlock covers multiple schools."
</blockquote></details>

Done.

**Review by pradh — APPROVED** [review]
Thanks, one last comment.

> *[Reviewable](https://reviewable.io/reviews/datacommonsorg/data/413#-:-MY8mswiDh9iX64syWSv:b-9re7tv)* status: 3 of 22 files reviewed, 11 unresolved discussions (waiting on @pradh and @thejeshgn)

___
*[scripts/india_udise/udise_geography/README.md, line 7 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6dbxuCoV0UqiA9OCS:-MY8l5TiAaGeaxtEc5Ko:b-ucfkn4) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/README.md#L7)):*
<details><summary><i>Previously, thejeshgn (Thejesh GN) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Thanks.
___
*[scripts/india_udise/udise_geography/UDISE_Blocks.tmcf, line 5 at r1](https://reviewable.io/reviews/datacommonsorg/data/413#-MY6awK_7rYSB9Dgygwu:-MY8m4oW2pWAnUXN7oWs:b-k08u4o) ([raw file](https://github.com/datacommonsorg/data/blob/3ad89daa127e29053e772d4f9c972d13a54d3ee8/scripts/india_udise/udise_geography/UDISE_Blocks.tmcf#L5)):*
<details><summary><i>Previously, thejeshgn (Thejesh GN) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

The column contents of udise_district_code should be "dcid:udiseCode/09" instead of "09" to indicate reference ?

(This is not needed for UDISE_Districts.tmcf because the column stores the value of udiseCode property of the state.)

**Comment by thejeshgn** [comment]
@pradh thanks for the note. Understood. Pushed the changes.

File: scripts/india_udise/udise_geography/UDISE_Blocks.csv [hunk 1]
```diff
@@ -0,0 +1,8192 @@
+udise_district_code,udise_block_code,block_name
```
Comment (pradh) [inline scripts/india_udise/udise_geography/UDISE_Blocks.csv:1]: does this file need to be updated?

Comment (thejeshgn) [reply scripts/india_udise/udise_geography/UDISE_Blocks.csv:1]: ✔️

**Pull request merged**